packages/flame/lib/src/game/mixins/single_game_instance.dart

```diff
@@ -0,0 +1,16 @@
+import '../../components/component.dart';
+import 'game.dart';
+
+mixin SingleGameInstance on Game {
```

Comment:
Would be good to have this documented in the docs (and with dartdocs)

Reply:
done